_data/baic/schedule/fall22.yml

```diff
@@ -0,0 +1,50 @@
+- abstract: "Biweekly Antiracism and Inclusion Conversations Joint Kick-off (Staff, Faculty and Grad Students)"
+  date: 2022-09-09
+  summary: Coming soon
+
+- abstract: "The Journey to Inclusive Cultures 101"
+  date: 2022-09-23
+  time: ""
```

Comment:
Time?

Reply:
I have made space for it, but since it is empty, it won't show up on the site. 
I can add them when we know it is fixed.

Reply:
Removed for now.